apps/studio/src/components/CmsSidebar/CmsSideNav.tsx

```diff
@@ -0,0 +1,192 @@
+import { Suspense } from "react"
+import { useRouter } from "next/router"
+import {
+  Accordion,
+  AccordionButton,
+  AccordionIcon,
+  AccordionItem,
+  AccordionPanel,
+  Box,
+  Flex,
+  Skeleton,
+  Text,
+  useDisclosure,
+  VStack,
+} from "@chakra-ui/react"
+import { Menu } from "@opengovsg/design-system-react"
+import { ResourceType } from "@prisma/client"
+import { BiData, BiFile, BiFolder, BiHomeAlt } from "react-icons/bi"
+
+import { trpc } from "~/utils/trpc"
+
+export const CmsSideNav = () => {
+  const { onToggle } = useDisclosure()
```

Comment:
this does not seem to be needed; you are just calling a function that is never used?

Reply:
it's being used on the menu button but according to your suggestion below, it might not be needed.

Reply:
ok checked, it's not needed - removed